scripts/jenkins/Jenkinsfile.release

```diff
@@ -0,0 +1,48 @@
+#!groovy
+
+properties([
+    buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '50')),
+
+    parameters([
+        string(name: 'TAG',
+               defaultValue: 'v0.0.0',
+               description: 'CLUO tag to release'),
+    ])
+])
+
+
+node('docker') {
+    step([$class: 'WsCleanup'])
+
+    stage('scm') {
+        checkout scm: [
+            $class: 'GitSCM',
+            branches: [[name: params.TAG]],
+            extensions: [[$class: 'RelativeTargetDirectory',
+            relativeTargetDir: 'container-linux-update-operator'],
+            [$class: 'CleanBeforeCheckout']],
+            userRemoteConfigs: [[url: "https://github.com/coreos/container-linux-update-operator.git", name: 'origin']]
+        ]
+    }
+
+    stage('build') {
+        withCredentials([usernamePassword(credentialsId: 'quay-update-operator', passwordVariable: 'DOCKER_PASS', usernameVariable: 'DOCKER_USER')]) {
+            withEnv(["TAG=${params.TAG}"]) {
+                sh '''#!/bin/bash -ex
+# workspace docker config
+                    export HOME="${WORKSPACE}"
+                    set +x
```

Comment:
You shouldn't need to do this.  Jenkins automatically blocks credentials values in console output.